src/app/components/Header/Header.tsx

```diff
@@ -54,9 +55,12 @@ export const Header = () => {
         )}
         {!isDesktop && (
           <div
-            onClick={handleToggleMenu}
-            className={`z-100 ${isMenuOpen ? "translate-x-[0%]" : "translate-x-[100%]"} absolute left-0 top-0 h-screen w-screen bg-backdrop transition duration-300 ease-in-out`}
-          >
+          onClick={handleToggleMenu}
+          className={cn(
+            'z-100 absolute left-0 top-0 h-screen w-screen bg-backdrop transition duration-300 ease-in-out',
+            { 'translate-x-[0%]': isMenuOpen, 'translate-x-[100%]': !isMenuOpen }
```

Comment:
It can be simplified: `isMenuOpen ? 'translate-x-0 : translate-x-full'` without curly brackets around it.